kratos/utilities/curve_axis_intersection.h

```diff
@@ -151,7 +152,10 @@ namespace Kratos
 
             // initialize axes
             IndexType axis_index_1, axis_index_2;
-            double min_1, max_1, min_2, max_2;
+            double min_1 = std::numeric_limits<double>::max();
+            double max_1 = std::numeric_limits<double>::lowest();
+            double min_2 = std::numeric_limits<double>::max();
+            double max_2 = std::numeric_limits<double>::lowest();
```

Comment:
```suggestion
            double min_1, max_1, min_2, max_2 = 0;
```

Would this also remove the warning? Those variables are overwritten in any case.

Reply:
> Would this also remove the warning?

don't think so, plus it might even lead to wrong results

> Those variables are overwritten in any case.

no, you need to check the code in detail. That is why the compiler complains

Reply:
But with this i would initialize them with one.